src/main/resources/assets/configured/lang/pl_pl.json

```diff
@@ -0,0 +1,10 @@
+{
+    "configured.gui.search": "Szukaj...",
```

Comment:
Unfortunately I haven't been able to find where this line is displayed in the game. Could you please provide me more context?

Reply:
It's currently unused but it's going to be placed in the search bar soon

Reply:
Okay, thank you for clarification. Now I'm sure I translated it correctly.